src/components/TransferDescription/index.js

```diff
@@ -178,7 +178,7 @@ class TransferDescription extends React.PureComponent {
           </Row>
           <Row>
             <TransferDescriptionItem
-              main={<SelectableText><NumericText value={assetBalanceAfter.amount.toString()} /> {assetToSend.symbol}</SelectableText>}
+              main={<SelectableText><NumericText maxDecimalPlaces={18} value={assetBalanceAfter.amount.toString()} /> {assetToSend.symbol}</SelectableText>}
```

Comment:
My React skills are a bit rusty, but is it necessary to use curly brackets for static values like this?

Reply:
For the number values, it needs to be wrapped with {}, but it is not needed for string values. Otherwise, the JSX parse throws error.